### ISSUE TITLE
Default response isn't set as the OperationResponse type

### ISSUE THREAD

**Issue by kiikka** [anttiviljami/openapicmd#59]
Hello!

I'm generating a type file for the client and I noticed that a response type "default" isn't set as the OperationResponse type in the generated file. It is always set as "any". In my case default is the only one I have defined in the yaml. 

**Yaml**

```yaml
/things:
    get:
      tags:
        - Things
      description: Get Things
      operationId: getThings
      responses:
        default:
          description: default response
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/Things"
            application/xml:
              schema:
                $ref: "#/components/schemas/Things"`
```

**openapi.d.ts**

```ts
namespace GetThings {
        namespace Responses {
            export type Default = Components.Schemas.Things;
        }
    }
```

```ts
'getThings'(
    parameters?: Parameters<UnknownParamsObject> | null,
    data?: any,
    config?: AxiosRequestConfig
  ): OperationResponse<any>
```

Is this intentional behaviour that default responses aren't set as the OperationResponse's type.

- Aleksi

**Comment by anttiviljami** [comment]
Nice catch, Aleksi! 👍

Fix available in `openapicmd@2.6.1`

**Comment by kiikka** [comment]
Thanks for the quick fix!
